src/theme/components/card.ts

```diff
@@ -14,23 +14,24 @@ const getBodyPaddingStyles = ({
 }) => {
   let bodyPadding = null;
   if (isBordered) {
-    bodyPadding = "s";
+    bodyPadding = "inset.default";
     if (hasImage) {
-      bodyPadding = "0 var(--nypl-space-s) var(--nypl-space-s)";
+      bodyPadding =
+        "0 var(--nypl-space-inset-default) var(--nypl-space-inset-default)";
     }
   }
   if (isRow && isBordered) {
-    bodyPadding = "var(--nypl-space-s)";
+    bodyPadding = "var(--nypl-space-inset-default)";
```

Comment:
I think this can be just `inset.default` like on line 17 above. The other `bodyPadding` have the CSS var syntax because there are multiple values.